### PR TITLE
feat: added logout button

### PR DESCRIPTION
**Pull request by harshalini**
Added logout button to wishlist and cart.
Screenshot (357)
Screenshot (358)

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This looks fine you can merge

**Review by paarthbajaj — APPROVED** [review]
Looks good. You can merge.

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -27,7 +27,30 @@
...
 }
-
+.logout-btn {
+    color: var(--orange);
+    border: solid 2px var(--orange);
+    background-color: var(--white);
+    font-weight: normal;
+    font-size: 1rem;
+    margin-top: 0.8rem;
+    margin-left: 1rem;
```
Comment (paarthbajaj) [inline wishlist/wishlist.css:37]: You can make utility class of "font-size: 1rem", "margin-left: 1 rem", etc. The reason for saying this is that these properties are the most common ones. You can make separate classes for them and use them whenever you want in any project. If you have already made them, then you could use them here.

Comment (harshalini) [reply wishlist/wishlist.css:37]: Ok, will take into consideration from now on, thanks for suggesting

**Pull request merged**